iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -350,6 +351,10 @@ public static boolean isCopyOnWriteMode(Context.Operation operation, BinaryOpera
     return RowLevelOperationMode.COPY_ON_WRITE.modeName().equalsIgnoreCase(mode);
   }
 
+  public static boolean isFanoutEnabled(Map<String, String> props) {
+    return PropertyUtil.propertyAsBoolean(props, "write.fanout.enabled", true);
```

Comment:
Is `write.fanout.enabled` a new property? Maybe consider creating a String constant.

Reply:
moved to InputFormatConfig.WRITE_FANOUT_ENABLED